.travis.yml

```diff
@@ -5,25 +5,23 @@ rvm:
   - 2.3.3
 cache: bundler
 
+env:
+  global:
+    - NOKOGIRI_USE_SYSTEM_LIBRARIES=true  # Speeds installation of html-proofer
```

Comment:
This doesnt need to be moved. It is yaml. Order is not important (99% of the time)

Reply:
I am sorry I misunderstood. Please tell which core repo are you referring to in your comment above ?

Reply:
https://github.com/coala/coala/

Reply:
I made this https://github.com/coala/coala/pull/5617  PR for this reason only.